torch/csrc/jit/runtime/register_special_ops.cpp

```diff
@@ -433,7 +433,6 @@ RegisterOperators reg({
         "aten::set_grad_enabled(bool val) -> ()",
         [](Stack* stack) {
           torch::GradMode::set_enabled(pop(stack).toBool());
-          push(stack, IValue());
```

Comment:
@eellison I remember we talked about this during the [`no_grad` PR](#41371):
<img width="878" alt="Captura de Pantalla 2020-10-06 a la(s) 10 17 36 a  m" src="https://user-images.githubusercontent.com/4392003/95237592-469f7200-07bd-11eb-923a-b66cb2addbcb.png">

I agree with the PR author's assessment that what we thought is the return value of `torch.set_grad_enabled(False)` is actually just Python printing out none where there is no return value but I wanted to double-check.

Reply:
Yea, i think my suggestion was you push None on the stack but we should have also updated the schema to reflect that

Reply:
No return value is also fine. I want to double check that the operator doesn't have a return value though when you annotate it with `()`. (printing the IR graph should show us)

Reply:
Confirmed 
```
graph():
  %2 : None = prim::Constant() # test/elias.py:5:0
  %0 : bool = prim::Constant[value=1]() # test/elias.py:6:27
   = aten::set_grad_enabled(%0) # test/elias.py:6:4
  return (%2)
```